### PR TITLE
Integrate WAL files to storage analysis

### PR DESCRIPTION
**Pull request by vmercierfr**
# Objective

Integrate WAL files to storage analysis 

# Why

This helps to have a better understanding of RDS instance storage usage

<details>
<summary>Example</summary>

WAL with annotation

</details>

# How

- Add `rds_transaction_log_disk_usage_byte` metrics (from AWS CloudWatch API)

# Release plan

- [ ] Merge this PR
- [ ] Apply with CI

### CODE REVIEW

**Comment by vmercierfr** [comment]
#sre

**Comment by vmercierfr** [comment]
#sre

**Review by qfritz — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -63,6 +63,7 @@ It collect key metrics about:
 | rds_quota_total_storage_bytes | `aws_account_id`, `aws_region` | Maximum total storage for all DB instances |
 | rds_read_iops_average | `aws_account_id`, `aws_region`, `dbidentifier` | Average number of disk read I/O operations per second |
 | rds_read_throughput_bytes | `aws_account_id`, `aws_region`, `dbidentifier` | Average number of bytes read from disk per second |
+| rds_transaction_log_disk_usage_bytes | `aws_account_id`, `aws_region`, `dbidentifier` | Disk space used by transaction logs (only on PostgreSQL) |
```
Comment (chtitux) [inline README.md:66]: Cloudwatch use `logs`, but we export the metric with `log`.

Is there a reason to skip the s when we export the metric, or should we use `logs` in the metric too?

```suggestion
| rds_transaction_logs_disk_usage_bytes | `aws_account_id`, `aws_region`, `dbidentifier` | Disk space used by transaction logs (only on PostgreSQL) |
```

File: internal/app/exporter/exporter.go [hunk 2]
```diff
@@ -235,6 +236,10 @@ func NewCollector(logger slog.Logger, collectorConfiguration Configuration, awsA
 			"Number of active sessions where the wait event type is not CPU",
 			[]string{"aws_account_id", "aws_region", "dbidentifier"}, nil,
 		),
+		transactionLogsDiskUsage: prometheus.NewDesc("rds_transaction_log_disk_usage_bytes",
```
Comment (chtitux) [inline internal/app/exporter/exporter.go:239]: Refer to `logs` vs `log` comment:

```suggestion
		transactionLogsDiskUsage: prometheus.NewDesc("rds_transaction_logs_disk_usage_bytes",
```

**Review by chtitux — APPROVED** [review]
I've added a comment about naming the new metric.

Otherwise, LGTM

File: README.md (hunk 1 above)
Comment (vmercierfr) [reply README.md]: Correct! I updated it

File: internal/app/exporter/exporter.go (hunk 2 above)
Comment (vmercierfr) [reply internal/app/exporter/exporter.go]: Correct! I updated it

**Pull request merged**